CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 # Build protocol buffers just to check they compile without errors.
 
 cmake_minimum_required(VERSION 2.8)
+add_definitions(-std=c++11)
```

Comment:
:-/

It's OK, it's just that there's a better way. This doesn't work with all compilers but we still don't care about those anyway.